### ISSUE TITLE
Make an option for a Spawn world where it uses your last gamemode's inventory

### ISSUE THREAD

**Issue by hyperion1008** [BentoBoxWorld/InvSwitcher#35]
### Is your feature request related to a problem?

I have a server with 2 gamemodes and a shared spawn, BSkyBlock and AOneblock. In the spawn world, there are some shops that both oneblock players and skyblock players can use, and every player can have both types of islands

### Describe the solution you'd like.

It would be great if I could set the Spawn world as an overlap world, where if I enter that world, and I have my BSkyBlock inventory, that is the inventory that is used, and if I was just in AOneBlock, this is the inventory that I keep.

### Describe alternatives you've considered.

I tried Multiverse-Inventories, but they also do not have this feature

### Agreements

- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] I have ensured the feature I'm requesting isn't already in the latest supported BentoBox build or Addons.

**Comment by tastybento** [comment]
Hmm, in your example, what would prevent the user from going from BSkyBlock to the spawn world and then to AOneBlock from there? Wouldn't the inventory just move over? How would you expect that to work?

Right now, there is a concept of world grouping in the code, e.g., the overworld, nether, and end are grouped and share the same inventory, etc. That could potentially be extended to include other worlds, and if there was a spawn world for each game separate then it could operate, but I can't work out how a shared spawn world would work. Wouldn't you also have issues with the economy? Or how about players from one world trading informally with the other world? Just dropping items, or stuff?

In a nutshell, I can imagine having a spawn world that shares inventory with BSkyBlock, and another spawn world that shares inventory with AOneBlock, but if you overlap the spawn world, then essentially all three worlds share the same inventory.

**Comment by tastybento** [comment]
I just read your title and you are suggesting that it's the last world's inventory that is used. So...

BSkyBlock world -> spawn world = keep BSkyBlock world inventory
AOneBlock world -> spawn world = keep AOneBlock world inventory

spawn world -> BSkyBlock world = use BSkyBlock world inventory
spawn world -> AOneBlock world = use AOneBlock world inventory

BSkyBlock world -> spawn world -> AOneBlock world = switch to AOneBlock world inventory
AOneBlock world -> spawn world -> BSkyBlock world = switch to BSkyBlock world inventory

AOneBlock -> BSkyBlock world = switch to BSkyBlock world inventory
BSkyBlock -> AOneBlock world = switch to AOneBlock world inventory

**Comment by hyperion1008** [comment]
That would be perfect, thank you. Sorry if I wasn't exactly clear on how it worked
